### ISSUE TITLE
dependencies without conda-forge

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#38]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/38) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Since Jörn has removed geopy we should be able to build without conda-forge

We have to find a setup of meta.yaml which works with win64 and linux64

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

This works on linux64


```
#!yaml

requirements:
  build:
    - basemap ==1.0.7
    - chameleon ==2.24
    - curl ==7.45.0
    - cycler ==0.10.0
    - execnet ==1.3.0
    - freetype ==2.5.5
    - functools32 ==3.2.3.2
    - hdf5 ==1.8.15.1
    - jpeg ==8d
    - libpng ==1.6.22
    - lxml ==3.6.0
    - matplotlib ==1.5.1
    - netcdf4 ==1.2.2
    - numpy ==1.11.1
    - paste ==1.7.5.1
    - pillow ==3.2.0
    - pyqt ==4.11.4
    - python ==2.7.12
    - python-dateutil ==2.5.3
    - pytz ==2016.4
    - qt ==4.8.7
    - requests ==2.10.0
    - scipy ==0.17.1
    - sip ==4.16.9
    - six ==1.10.0
    - tk ==8.5.18
  run:
    - basemap ==1.0.7
    - chameleon ==2.24
    - curl ==7.45.0
    - cycler ==0.10.0
    - execnet ==1.3.0
    - freetype ==2.5.5
    - functools32 ==3.2.3.2
    - hdf5 ==1.8.15.1
    - jpeg ==8d
    - libpng ==1.6.22
    - lxml ==3.6.0
    - matplotlib ==1.5.1
    - netcdf4 ==1.2.2
    - numpy ==1.11.1
    - paste ==1.7.5.1
    - pillow ==3.2.0
    - pyqt ==4.11.4
    - python ==2.7.12
    - python-dateutil ==2.5.3
    - pytz ==2016.4
    - qt ==4.8.7
    - requests ==2.10.0
    - scipy ==0.17.1
    - sip ==4.16.9
    - six ==1.10.0
    - tk ==8.5.18

```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

resolved by commit #2c08ac9
